packages/effector/index.d.ts

```diff
@@ -1701,7 +1701,7 @@ type SampleFilterTargetDef<
         : [message: {error: 'function should accept data source types'; got: FN}]
     : FLBool extends BooleanConstructor
       ? Mode extends Mode_Flt_Fn_Trg
-        ? FNAltArg extends (arg: infer FNArg) => any
+        ? FNAltArg extends (arg?: any, clk?: any) => any
```

Comment:
Great trick, need to try to use it in other places in types, it should be much easier to use than checking functions case-by-case